Otel Collector tags information added (#5050)
SumoLogic/sumologic-documentation: docs/send-data/opentelemetry-collector/remote-management/source-templates/manage-source-templates.md

1. From the **Source Template** page, select the source template that you need to delete.
1. Click the **Delete** button (or use the kebab menu against the selected source template, click **Delete** from the dropdown).
1. Confirm the deletion. The source template will be removed from the **Source Template** page and unlinked from all collectors.

## Collector tags

With remote management, you can tag your [OpenTelemetry Collectors](/docs/send-data/opentelemetry-collector) to categorize and group them. These tags are also enriched in your data, enabling you to use them in your dashboards and searches.

:::note
- Collector tags will only be editable for collectors with version `0.114.0` and above.
- For default collector tags, you can only edit the **Values**.

> - System collector tags like `sumo.disco.enabled` cannot be added/updated/deleted.

Reviewer: ```suggestion
- System collector tags like `sumo.disco.enabled` cannot be added, updated, or deleted.
```

:::

You can add collector tags during OTEL collector setup as described in the previous section. Follow the below steps to add/update/delete the collector tags after creation:

1. Navigate to the **OpenTelemetry Collection** page.<br/><img src={useBaseUrl('img/send-data/opentelemetry-collection.png')} alt="opentelemetry-collection" style={{border:'1px solid gray'}} width="700"/>
1. Click on the desired collector where you need to add collector tags.
1. In the right pane, scroll to the **Collector Tags** section and click **Edit**.<br/><img src={useBaseUrl('img/send-data/collector-tags-edit.png')} alt="collector-tags-edit" style={{border:'1px solid gray'}} width="500"/>

> 1. In the edit window, click **+ Add tags** to add new tag or Update/Delete the required tags and click **Proceed**.<br/><img src={useBaseUrl('img/send-data/add-delete-update-tags.png')} alt="add-delete-update-tags" style={{border:'1px solid gray'}} width="500"/>

Reviewer: ```suggestion
1. In the edit window, click **+ Add tags** to add a new tag. You can also update or delete the required tags. Then click **Proceed**.<br/><img src={useBaseUrl('img/send-data/add-delete-update-tags.png')} alt="add-delete-update-tags" style={{border:'1px solid gray'}} width="500"/>
```

1. A new **Update Collector Tag(s)** window appears. Click the **click here** link to understand the impact of this update.<br/><img src={useBaseUrl('img/send-data/update-tags-click-here.png')} alt="update-tags-click-here" style={{border:'1px solid gray'}} width="500"/>
1. You will be redirected to the **Mapped Source Template** section, which displays the mapped source templates for the collector tags added. This window also displays the **Compatible** and **Incompatible Version** details for the mapped source template for the collector.<br/><img src={useBaseUrl('img/send-data/mapped-source-templates.png')} alt="mapped-source-templates" style={{border:'1px solid gray'}} width="500"/>
1. If there are any incompatible source templates, [upgrade the source template](/docs/send-data/opentelemetry-collector/remote-management/source-templates/manage-source-templates/#upgrade-a-source-template) to ensure compatibility.

> 1. Once the source template is upgraded and is compatible, enter **PROCEED** and click **Confirm** to add/update/delete the new collector tag.<br/><img src={useBaseUrl('img/send-data/confirm-the-tags.png')} alt="confirm-the-tags" style={{border:'1px solid gray'}} width="500"/>

Reviewer: ```suggestion
1. Once the source template is upgraded and is compatible, enter **PROCEED** and click **Confirm**.<br/><img src={useBaseUrl('img/send-data/confirm-the-tags.png')} alt="confirm-the-tags" style={{border:'1px solid gray'}} width="500"/>
```

1. A new **Update Collector Tag(s)** window appears. Click the **click here** link to understand the impact of this update.<br/><img src={useBaseUrl('img/send-data/update-tags-click-here.png')} alt="update-tags-click-here" style={{border:'1px solid gray'}} width="500"/>
1. You will be redirected to the **Mapped Source Template** section, which displays the mapped source templates for the collector tags added. This window also displays the **Compatible** and **Incompatible Version** details for the mapped source template for the collector.<br/><img src={useBaseUrl('img/send-data/mapped-source-templates.png')} alt="mapped-source-templates" style={{border:'1px solid gray'}} width="500"/>
1. If there are any incompatible source templates, [upgrade the source template](/docs/send-data/opentelemetry-collector/remote-management/source-templates/manage-source-templates/#upgrade-a-source-template) to ensure compatibility.
1. Once the source template is upgraded and is compatible, enter **PROCEED** and click **Confirm**.<br/><img src={useBaseUrl('img/send-data/confirm-the-tags.png')} alt="confirm-the-tags" style={{border:'1px solid gray'}} width="500"/>

:::info

> If you try to add/update any collector tags using the API, the following validations will occur in the backend:

Author: ```suggestion
If you try to add or update any collector tags using the API, the following validations will occur in the backend:
```

1. Validates if the collector is remotely managed.
1. Validates if the key/value is empty string.
1. Validates the agent version.

> 1. Validates if the disco tags are added/deleted/updated.

Author: ```suggestion
1. Validates if the disco tags are added, updated, or deleted.
```
